### PR TITLE
[1] Small Test Fixes

### PR DESCRIPTION
**Pull request by IDragonfire**

### CODE REVIEW

**Comment by IDragonfire** [comment]
@bukajsytlos is this Pr fine? Can you then accept the review?

File: src/test/java/com/faforever/api/data/JsonApiClanIntegrationTest.java [hunk 1]
```diff
@@ -161,14 +163,10 @@ public void getFilteredPlayerForClanInvite() throws Exception {
...
-    JsonNode node = objectMapper.readTree(action.andReturn().getResponse().getContentAsString());
-
-    assertEquals(2, node.get("data").size());
-    assertEquals(players[1], node.get("data").get(0).get("attributes").get("login").asText());
-    assertEquals(players[0], node.get("data").get(1).get("attributes").get("login").asText());
-
-    action.andExpect(status().isOk());
+    action.andExpect(status().isOk())
+      .andExpect(jsonPath("$.data", hasSize(2)))
+      .andExpect(jsonPath("$.data[0].attributes.login", is("DRAGON")))
```
Comment (IDragonfire) [inline src/test/java/com/faforever/api/data/JsonApiClanIntegrationTest.java]: yes `.perform(get("/data/player?filter=login==dragon*&sort=login"))`

Comment (IDragonfire) [inline src/test/java/com/faforever/api/data/JsonApiClanIntegrationTest.java]: yes `.perform(get("/data/player?filter=login==dragon*&sort=login"))`

Comment (bukajsytlos) [inline src/test/java/com/faforever/api/data/JsonApiClanIntegrationTest.java]: ok

**Comment by micheljung** [comment]
After conflicts are resolved, I'll merge this

File: src/test/java/com/faforever/integration/factories/SessionFactory.java [hunk 2]
```diff
@@ -25,14 +25,14 @@
   private static final String OAUTH_CLIENT_ID = "1234";
   private static final String OAUTH_SECRET = "secret";
 
-  private static ObjectMapper objectMapper = new ObjectMapper();
-  private static ShaPasswordEncoder shaPasswordEncoder = new ShaPasswordEncoder(256);
+  private static final ObjectMapper objectMapper = new ObjectMapper();
```
Comment (micheljung) [inline src/test/java/com/faforever/integration/factories/SessionFactory.java]: Whenever you're instantiating an `ObjectMapper` you're doing it wrong ;) inject it from outside, or add it as method parameter

**Comment by IDragonfire** [comment]
@micheljung conflict resolved

**Pull request merged**